recordedit/index.html.in

```diff
@@ -279,8 +282,11 @@
                         </div>
                     </div>
                 </div>
+
             </div>
         </div>
+
     </div>
+    <footer ng-show="displayReady"></footer>
```

Comment:
Same comment as below. I don't think there is a reason to wait for the app to load before showing the footer.